.github/workflows/industrial_ci_action.yml

```diff
@@ -10,15 +10,22 @@ jobs:
     strategy:
       matrix:
         env:
-          - {ROS_DISTRO: melodic, ROS_REPO: main, CATKIN_LINT: pedatntic}
           - {
-              ROS_DISTRO: kinetic, ROS_REPO: main, 
-              BEFORE_BUILD_TARGET_WORKSPACE: './.ci.before_script', 
-              ROSDEP_SKIP_KEYS: 'libmongoclient-dev',
-              ADDITIONAL_DEBS: 'libssl-dev scons git'
-            }
+            ROS_DISTRO: melodic, ROS_REPO: main, CCOV_UPLOAD: true, 
+            CATKIN_LINT: pedatntic,
+            CMAKE_ARGS: "-DCMAKE_BUILD_TYPE=Debug -DCMAKE_C_FLAGS='--coverage' -DCMAKE_CXX_FLAGS='--coverage'",
+            AFTER_RUN_TARGET_TEST: './.ci.prepare_codecov',
+            ADDITIONAL_DEBS: 'lcov'
+          }
+          - {
+            ROS_DISTRO: kinetic, ROS_REPO: main, CCOV_UPLOAD: false, 
+            BEFORE_BUILD_TARGET_WORKSPACE: './.ci.mongo_source_install', 
+            ROSDEP_SKIP_KEYS: 'libmongoclient-dev',
+            ADDITIONAL_DEBS: 'libssl-dev scons git'
+          }
```

Comment:
Please reformat as follows (each dict entry should have its own line):
```yaml
        env:
          - {
              ROS_DISTRO: melodic,
              ROS_REPO: main,
              CCOV_UPLOAD: true,
              CATKIN_LINT: pedatntic,
              CMAKE_ARGS: "-DCMAKE_BUILD_TYPE=Debug -DCMAKE_C_FLAGS='--coverage' -DCMAKE_CXX_FLAGS='--coverage'",
              AFTER_RUN_TARGET_TEST: "./.ci.prepare_codecov",
              ADDITIONAL_DEBS: "lcov",
            }
          - {
              ROS_DISTRO: kinetic,
              ROS_REPO: main,
              CCOV_UPLOAD: false,
              BEFORE_BUILD_TARGET_WORKSPACE: "./.ci.mongo_source_install",
              ROSDEP_SKIP_KEYS: "libmongoclient-dev",
              ADDITIONAL_DEBS: "libssl-dev scons git",
            }
```

Reply:
I did this and brought in the change of the cache prefix.  I really like how that simplifies the cache keys.